### ISSUE TITLE
update the layer list

### ISSUE THREAD

**Issue by 12rambau** [12rambau/restoration_planning_module#18]
In the layer list things are missing : 
- description of the layer : it should include both the description and the technical input to reproduce it locally
- names : some name are way too long to be displayed in the proposed dashboard, do you think we could shorten them
- layers : lots of them are out of date if not fake
- the other subtheme is not referenced in the wireframe so I assume it has been abandoned

In short if you have an updated google sheet I would be happy to update the csv file

**Comment by jdilger** [comment]
Spoke with Jeff and Karis about shorting the layer names and they're happy to do it for us. Is there a character limit or ideal range I can give them/us to work with?

**Comment by jdilger** [comment]
I think we're using the subthemes for assigning the weights via the priorities panel (at least for now) so I say we keep them in.

**Comment by 12rambau** [comment]
> Spoke with Jeff and Karis about shorting the layer names and they're happy to do it for us. Is there a character limit or ideal range I can give them/us to work with?

between 30 and 40 character is a must. In fact I canot guarantee that all the letters will be displayed in all conditions after 30. but for big screens you can go to 40
